Added the missing polygon value

Sojaner opened the pull request:
According to the documentation, there are 2 types of boundary which, one is missing from the types.

Sojaner committed:
Added the missing polygon value

i8beef commented:
On the `Boundary` type there is also a new field apparently called `Relevancy` which should be added as a `List<decimal>` with description denoting the following usage:

```
This element MUST be present only if type is polygon. It gives the coordinates of a point inside the polygon with format [x,y] where x is the horizontal axis and y the vertical axis expressed as the fraction of respectively the width and the height of the map.

A polygon may intersect walls, and hence cover areas of different and eventually non-adjacent rooms. relevancy is the point that defines the area of interest as the set of all floor points that are contiguous to this point and are contained in the polygon.
```

i8beef commented:
Did you get a chance to update the above? I'd love to merge this in.

Sojaner committed:
Added the Relevancy property to the Boundary type.

Sojaner commented:
Here you go.

I have added the Relevancy property to the Boundary class and added the description as the summary documentation of the property.

Feel free to merge when you see everything is correct.
